### PR TITLE
LPS 180100

### PR DESCRIPTION
**Pull request by robertoDiaz**
Thank you @ambrinchaudhary for the help!

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/bbcode/bbcode_parser.js [hunk 1]
```diff
@@ -628,7 +628,10 @@
 
 			const imageSrcInput = instance._extractData(STR_IMG, true);
 
-			if (REGEX_IMAGE_SRC.test(imageSrcInput)) {
+			if (
+				REGEX_IMAGE_SRC.test(imageSrcInput) ||
+				imageSrcInput.startsWith('data:image/')
```
Comment (adolfopa) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/bbcode/bbcode_parser.js:633]: Can you reverse the test? `startsWith` should be much faster than a regex test.
```
if (imageSrcInput.startsWith(...) || REGEX_IMAGE_SRC.test(...))
```

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/bbcode/bbcode_parser.js (hunk 1 above)
Comment (robertoDiaz) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/bbcode/bbcode_parser.js:633]: i'll do it in both cases :)

Comment (robertoDiaz) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/bbcode/bbcode_parser.js:633]: yes, I'll do it in both cases

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
The functional test failure is not related to this PR.

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by Hanlf** [comment]
The above failure is not related to this PR, forward it to BChan.

**Pull request closed without merging**